models/event.py

```diff
@@ -47,6 +47,7 @@ class Event(ndb.Model):
     facebook_eid = ndb.StringProperty(indexed=False)  # from Facebook
```

Comment:
`facebook_eid` is already present for this -- there's no need to add `facebook_event`. EID = Event ID.